packages/plugins/plugin-markdown/src/MarkdownPlugin.tsx

```diff
@@ -256,7 +256,14 @@ export const MarkdownPlugin = (): PluginDefinition<MarkdownPluginProvides> => {
             return range?.start ?? Number.MAX_SAFE_INTEGER;
           };
 
-          return (anchorA: string, anchorB: string) => getStartPosition(anchorA) - getStartPosition(anchorB);
+          return (anchorA: string | undefined, anchorB: string | undefined): number => {
```

Comment:
`predicate: (obj) => obj instanceof DocumentType,`
above doesn't match prototype (should return object not boolean)

Reply:
Can you elaborate on this? It's currently working as intended.

Reply:
```
The `obj is T` type predicate in the `predicate` function:

1. Indicates a runtime type check
2. Narrows the type of `obj` to `T` if the function returns true
```